src/screen.rs

```diff
@@ -67,6 +67,7 @@ pub enum ScreenInstruction {
     ClearScroll,
     CloseFocusedPane,
     ToggleActiveTerminalFullscreen,
+    #[allow(dead_code)]
```

Comment:
I'm not sure about adding warning suppressions (similarly for the other allow directives)

Reply:
I feel the same... @5c077m4n - could you share the reasoning here?

Reply:
ah, oops - didn't realize this was already changed